ExpoDemo/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,45 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  <uses-permission android:name="android.permission.INTERNET"/>
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+  <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+  <uses-permission android:name="android.permission.VIBRATE"/>
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+  <queries>
+    <intent>
+      <action android:name="android.intent.action.VIEW"/>
+      <category android:name="android.intent.category.BROWSABLE"/>
+      <data android:scheme="https"/>
+    </intent>
+  </queries>
+  <application android:name=".MainApplication" android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:allowBackup="true" android:theme="@style/AppTheme" android:supportsRtl="true">
+    <service
+      android:name="com.raygun.raygun4android.services.CrashReportingPostService"
+      android:exported="false"
+      android:permission="android.permission.BIND_JOB_SERVICE"
+      android:process=":crashreportingpostservice"
+    />
+    <service
+      android:name="com.raygun.raygun4android.services.RUMPostService"
+      android:exported="false"
+      android:permission="android.permission.BIND_JOB_SERVICE"
+      android:process=":rumpostservice"
+    />
```

Comment:
Added the following lines to the default demo project code, to support Raygun native crash reporting, following the "additional steps for Android"

Reply:
Yes